indico/modules/events/client/js/reviewing/components/ChangesConfirmation.jsx

```diff
@@ -0,0 +1,70 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React from 'react';
+import {Form as FinalForm} from 'react-final-form';
+import {Divider, Form, Message} from 'semantic-ui-react';
+import {FinalSubmitButton, FinalTextArea} from 'indico/react/forms';
+import {Translate} from 'indico/react/i18n';
+
+import './ChangesConfirmation.module.scss';
+
+export default function ChangesConfirmation() {
+  const confirmChanges = formData => {
+    console.log(formData);
```

Comment:
we already have the APIs for this so let's actually implement answering. can probably use a redux action similar to what we use when submitting a review